frontend/resourceadm/pages/DeployResourcePage/DeployResourcePage.tsx

```diff
@@ -132,9 +99,9 @@ export const DeployResourcePage = ({ navigateToPageWithError }: Props) => {
    * Returns the correct error type for the deploy page
    */
   const getStatusError = (): DeployErrorType[] | string => {
-    if (hasResourceError || hasPolicyError !== 'none') {
+    if (validateResourceData.status !== 200 || hasPolicyError !== 'none') {
       const errorList: DeployErrorType[] = [];
-      if (hasResourceError) {
+      if (validateResourceData.status !== 200) {
```

Comment:
As mentioned above, the status checks is on the actual object returned from backend. It can be 400 even if it is in the success (200) part of the API Call.